src/components/HelpDrawer/HelpContent/SpectralProfilerHelpContent.tsx

```diff
@@ -81,37 +89,39 @@ export const SPECTRAL_PROFILER_HELP_CONTENT = (
             <li>Air wavelength (m, mm, um, Angstrom)</li>
             <li>Channel</li>
         </ul>
-        <p>and spectral reference frame, including:</p>
+        <p>the spectral reference frame, including:</p>
         <ul>
             <li>LSRK: the rest-frame of the kinematical local standard of rest</li>
             <li>LSRD: the rest-frame of the dynamical local standard of rest</li>
             <li>BARY: barycentric, the rest-frame of the solar-system barycenter</li>
             <li>TOPO: topocentric, the observer's rest-frame on Earth</li>
         </ul>
+        <p>and the intensity unit such as &quot;Jy/beam&quot; &lt;-&gt; &quot;K&quot;, depending on the integrity image header.</p>
```

Comment:
The integrity of the image header? The contents of the image header? Not sure about this.